content/articles/2018-12-textrecipes-0-0-1.Rmarkdown

```diff
@@ -33,13 +33,13 @@ install.packages("textrecipes")
 
 ## New steps
 
-The steps introduces here can be split into 3 types. The steps
+The steps introduced here can be split into 3 types, those that:
 
-1. converts from characters to list-columns and vice versa
-1. modifies the elements in list-columns
-1. converts list-columns to numerics
+1. convert from characters to list-columns and vice versa,
+1. modify the elements in list-columns, and
+1. convert list-columns to numerics.
 
-this allows for great flexibility in the preprocessing tasks that can be done while staying inside the [`recipes`](https://github.com/tidymodels/recipes) framework. This also prevents having one single step having a million arguments.
+This allows for greater flexibility in the preprocessing tasks that can be done while staying inside the [recipes](https://github.com/tidymodels/recipes) framework. This also prevents having one single step with a many arguments.
```

Comment:
Nix the "a" between with and many (so it reads "with many arguments.")

Reply:
Actually, phrasing suggestion for the last sentence:
"This also prevents having a single step with many arguments."